index.html

```diff
@@ -184,5 +264,102 @@ <h4 class="top-banner__subtitle">
         </div>
       </div>
     </section>
+    <section class="why-us">
+      <div class="why-us__container">
+        <div class="why-us__main-content">
+          <h2 class="why-us__title">WHY CHOOSE US</h2>
```

Comment:
Don't write text in capital letters in HTML. Always imagine that you don't have access to the text content. It could come from server or somebody can edit it and will use lowercase for example. In this case the design will be broken. So use CSS instead to make it uppercse.